lib/api/fragments/educator-page.js

```diff
@@ -38,11 +38,6 @@ fragment educatorPageFragmentFull on pages_educatorPages_Entry {
         ${getImageFields("crop", 900, 550)}
       }
     }
-    openGraphImage: hero {
```

Comment:
opengraph images are fetched from a single metadata method instead of as part of the entry's query